### ISSUE TITLE
Visual bug in entry widget

### ISSUE THREAD

**Issue by IshanJ25** [TomSchimansky/CustomTkinter#165]
how to fix this:

![image](https://user-images.githubusercontent.com/86649457/172434977-67bce9fd-ca09-47fb-affa-b7654460fd7d.png)

here is my piece of code.

```python
    entry = ctk.CTkEntry(master=login_frame,
                         placeholder_text="CTkEntry",
                         width=120,
                         height=25,
                         border_width=2,
                         corner_radius=10,
                         text_font='inter-medium 20 normal'
                         )
    entry.place(relx=0.5, rely=0.5, anchor='center')
```

when i put the font argument, the entry widgets develops a visual bug at corners

**Comment by Think-IT-think-gui** [comment]
the thing is, the entry box itself is our a widget of old tkinter, that has been placed in another resizable widget that has a modern entry box design in it, so when you increase the test font, the actual text box becomes bigger than the designed entry box.. My advice, keep the font as how it is..

**Comment by pourhakimi** [comment]
If you set `border_width=0` then the border is no longer visible and subsequently the edges are not visible either. You can even increase the font and the edges should not show (tested on macOS).

**Comment by TomSchimansky** [comment]
It's a bug. It will be fixed in the next version.

**Comment by TomSchimansky** [comment]
This should be fixed now with version 4.4.0.
